src/components/layout/layout.js

```diff
@@ -56,6 +73,12 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
         }
     }, [])
 
+    React.useEffect(() => {
+        LC_API.on_after_load = () => {
+            setLiveChatInteractive(true)
+        }
+    }, [LC_API])
```

Comment:
is this effect triggered when `isBrowser() && window.LC_API` are changed but there is no render?

Reply:
sort of, its triggered when LC_API changed

Reply:
But LC_API changing won't trigger a rerender I think?

Reply:
yes, but `setLiveChatInteractive` does

Reply:
but that won't be fired unless the hook is triggered, what about using https://developers.livechat.com/docs/extending-ui/extending-chat-widget/javascript-api/#on-ready after the render []?